### ISSUE TITLE
Edited samples to remove the initial silence before most samples

### ISSUE THREAD

**Issue by essej** [endolith/Salamander-Drumkit#2]
It turns out the original samples for this kit available in the archives almost all have pretty sizable blocks of silent audio at the sample starts, causing the use of the instruments to have unnecessary and very noticeable latency.

I went through all the samples and trimmed off this unnecessary silence and the playability is *way* better, especially with e-drum kits.

I've put my modified samples in a ZIP here for now:
   https://drive.google.com/file/d/1K1gbuUhSqlMem2sZI48NHSrqdJ6Uheh8/view?usp=sharing

You are welcome to take those and put them somewhere more appropriate, or you can post a link in the README.md.

**Comment by endolith** [comment]
I don't remember the details about the different versions, etc.  Can you submit a pull request linking your samples in the readme?  The silent audio can't be skipped by modifying the sfz files?

**Comment by essej** [comment]
It could have been handled with a different sample count start offset in every SFZ region, but it would have been a *tremendous* pain to do, and only result in more issues down the line, in subsequent editing of SFZ files that use those samples. In my view the original files are effectively faulty and there is no reason to keep the dead air in there, especially in drum samples. I'm just surprised no one caught this sooner!

I can do a PR with the README updated....

**Comment by endolith** [comment]
I vaguely remember wanting to do the same but then seeing that the problem is fixed in kinwie's version?  Does that make sense?  I could be remembering wrong

**Comment by essej** [comment]
No, Kinwie's version just fixes the end points of the kick samples due to some lack of fading there. The issue I discovered affects the onsets of almost all the samples, so I decided to simply edit them all. In fact, since I realized I wasn't actually using kinwie's version, but my own edit (oops!), I also fixed these end fade issues in my samples as well, so it may actually require some mods to kinwie's kick sample regions (removing the end opcodes) if you want to use my new samples with it...

**Comment by essej** [comment]
Edited PR by adding new sfz to use with my samples, removing the fixes kinwie did to the end sample offsets for the faulty original samples.
